Mathlib/Data/ENNReal/Holder.lean

```diff
@@ -104,12 +111,24 @@ lemma inv_sub_inv_eq_inv' (hq : q ≠ 0) : r⁻¹ - q⁻¹ = p⁻¹ := by
     simp_all
   · exact inv_sub_inv_eq_inv p q hr.ne'
 
+variable {r} in
+lemma unique_of_ne_zero (q' : ℝ≥0∞) (hr : r ≠ 0) [HolderTriple p q' r] : q = q' := by
+  rw [← inv_inj, ← inv_sub_inv_eq_inv q p hr, ← inv_sub_inv_eq_inv q' p hr]
+
 end HolderTriple
 
 /-! ### Hölder conjugates -/
 
 namespace HolderConjugate
 
+/- This instance causes a trivial loop, but this is exactly the kind of loop that
+Lean should be able to detect and avoid.
+
+I'm not sure we actually want this, but the hope is that it allows Lean to find this instead of
+unfolding? -/
+instance symm {p q : ℝ≥0∞} [hpq : HolderConjugate p q] : HolderConjugate q p :=
+  inferInstance
```

Comment:
That's the only thing I am not sure about in this PR. Everything else looks good.
How can we tell if this is problematic? Bench the PR?